js/index.js

```diff
@@ -0,0 +1,38 @@
+import { SlideDeck } from './slidedeck.js';
+import L from 'leaflet';
```

Comment:
**suggestion:** In order to use Leaflet from an `import` like this, you have to be using a development server that knows how to deal with it, and you have to build your project to run on any server. This is similar to the `vite` tool that I showed in class on Wednesday. I'll be re-recording the second half of that class to show how to build a project that uses these imports to deploy to the web, but as long as you include your `<script` tag in the HTML (i.e., as you do [here](https://github.com/Weitzman-MUSA-JavaScript/story-map-project/pull/19/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R101)) then you don't need this import.
```suggestion
```